snakemake/deployment/singularity.py

```diff
@@ -76,7 +76,7 @@ def pull(self, dryrun=False):
                         "singularity",
                         "pull",
                         "--name",
-                        "{}.simg".format(self.hash),
+                        "{}.sif".format(self.hash),
```

Comment:
simg hasn't been around for years now, and most containers are sif, so I updated this.